docs/integration/OpenAPI.md

```diff
@@ -77,7 +77,7 @@ components {
 | namePattern            | false    | .*      | Regexp for filtering operations by operationId or by created service name (concatenation of HTTP method, path and parameters)                                                                                                                                    |
 | security               | false    |         | Configuration for [authentication](https://swagger.io/docs/specification/authentication/) for each `securitySchemas` defined in the *OpenAPI interface definition*                                                                                               |
 | security.*.type        | false    |         | Type of security configuration for a given security schema. Currently only `apiKey` is supported                                                                                                                                                                 |
-| security.*.apiKeyValue | false    |         | API key that will be passed into the service via header, query parameter or cookie (depending on definition provided in OpenAPI)                                                                                                                                 |
```

Comment:
I find it difficult to understand statement like "_depending on definition provided in OpenAPI_". 

OpenAPI is too vague for me, it can mean standard defitinition, a particular service (interface) definition or maybe something else. I prefer to be more helpful here.